### PR TITLE
[pipes] IS_DAGSTER_PIPED_PROCESS -> IS_DAGSTER_PIPES_PROCESS

### PR DESCRIPTION
**Pull request by alangenfeld**
We updated the var but not the value.

Can we even do this now that we're past 1.5.0 ?

If we are not aleady past the point of no return we better do this now. 

## How I Tested These Changes

bk

### CODE REVIEW

**Comment by alangenfeld** [comment]
Current dependencies on/for this PR:
* master
  * **PR #17072** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/17072" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/17072?utm_source=stack-comment).

File: python_modules/dagster-pipes/dagster_pipes/__init__.py [hunk 1]
```diff
@@ -67,7 +67,7 @@ def _make_message(method: str, params: Optional[Mapping[str, Any]]) -> "PipesMes
 
 # ##### PARAMETERS
 
-IS_DAGSTER_PIPES_PROCESS = "IS_DAGSTER_PIPED_PROCESS"
+IS_DAGSTER_PIPES_PROCESS = "IS_DAGSTER_PIPES_PROCESS"
```
Comment (alangenfeld) [inline python_modules/dagster-pipes/dagster_pipes/__init__.py:70]: noticed while working on rust demo where have to refer to real value

File: python_modules/dagster-pipes/dagster_pipes/__init__.py [hunk 2]
```diff
@@ -67,7 +67,7 @@ def _make_message(method: str, params: Optional[Mapping[str, Any]]) -> "PipesMes
...
 
 DAGSTER_PIPES_BOOTSTRAP_PARAM_NAMES = {
     k: _param_name_to_env_key(k) for k in (IS_DAGSTER_PIPES_PROCESS, "context", "messages")
```
Comment (alangenfeld) [inline python_modules/dagster-pipes/dagster_pipes/__init__.py:73]: and actually because of this its `DAGSTER_PIPES_IS_DAGSTER_PIPED_PROCESS`

**Review by schrockn — APPROVED** [review]
I consider this a bug that should be fixed, not a backwards compat issue.

**Pull request merged**